### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by TrySound**
No breaking changes

### CODE REVIEW

**Comment by TrySound** [comment]
@fb55 \cc

File: package.json [hunk 1]
```diff
@@ -32,19 +32,19 @@
...
-    "coveralls": "*",
-    "istanbul": "*",
-    "jscs": "1.5.8",
-    "jshint": "2"
+    "coveralls": "^2.11.3",
+    "istanbul": "^0.3.17",
+    "jscs": "^2.0.0",
+    "jshint": "^2.8.0",
+    "mocha": "^2.2.5",
+    "mocha-lcov-reporter": "0.0.2"
```
Comment (fb55) [inline package.json]: Why this specific version (and not `^`)?

Comment (TrySound) [inline package.json]: Hm.. don't remember. Will fix

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (fb55) [inline .gitignore]: I maintain a global .gitignore. Adding one to the repo should be a different discussion.

Comment (TrySound) [inline .gitignore]: I don't use global gitignore like many other people. It's okay for npmdebug or another single file. But node_modules is too big for git status and should be in every package. It's my opinion.

**Comment by TrySound** [comment]
\cc @fb55

**Comment by fb55** [comment]
Thanks!

**Pull request merged**